sdk/webpubsub/Azure.Messaging.WebPubSub/src/WebPubSubServiceClient.cs

```diff
@@ -0,0 +1,969 @@
+            }
+            catch (Exception e)
+            {
+                scope.Failed(e);
+                throw;
+            }
+        }
+
+        /// <summary>
+        /// Send content inside request body to a group of connections.
+        /// </summary>
+        /// <param name="group">Target group name, which length should be greater than 0 and less than 1025.</param>
+        /// <param name="message"></param>
+        /// <param name="contentType">Supported values are: application/json, application/octet-stream, and text/plain.</param>
+        /// <param name="excluded">Excluded connection Ids</param>
+        /// <param name="cancellationToken"></param>
+        /// <returns>A <see cref="Response"/> representing the result of the operation.</returns>
+        public virtual Response SendToGroup(string group, RequestContent message, string contentType = JsonContent, IEnumerable<string> excluded = null, CancellationToken cancellationToken = default)
+        {
+            using var scope = _clientDiagnostics.CreateScope("WebPubSubServiceRestClient.SendToGroup");
+            scope.Start();
+            try
+            {
+                return RestClient.SendToGroup(hub, group, message, contentType, excluded, cancellationToken);
+            }
+            catch (Exception e)
+            {
+                scope.Failed(e);
+                throw;
+            }
+        }
+
+        /// <summary>Check if the connection with the given connectionId exists</summary>
+        /// <param name="connectionId">The connection Id.</param>
+        /// <param name="cancellationToken"></param>
+        /// <returns>A <see cref="Task{TResult}"/> representing the result of the asynchronous operation.</returns>
+        public virtual async Task<Response<bool>> ConnectionExistsAsync(string connectionId, CancellationToken cancellationToken = default)
+        {
+            using var scope = _clientDiagnostics.CreateScope("WebPubSubServiceRestClient.CheckConnectionExistenceAsync");
+            scope.Start();
+            try
+            {
+                var response = await RestClient.ConnectionExistsAsync(hub, connectionId, cancellationToken).ConfigureAwait(false);
+                return Response.FromValue(response.Status == 200, response);
+            }
+            catch (Exception e)
+            {
+                scope.Failed(e);
+                throw;
+            }
+        }
+
+        /// <summary>Check if the connection with the given connectionId exists</summary>
+        /// <param name="connectionId">The connection Id.</param>
+        /// <param name="cancellationToken"></param>
+        /// <returns>A <see cref="Response"/> representing the result of the operation.</returns>
+        public virtual Response<bool> ConnectionExists(string connectionId, CancellationToken cancellationToken = default)
+        {
+            using var scope = _clientDiagnostics.CreateScope("WebPubSubServiceRestClient.SendToGroup");
```

Comment:
In case you copied all these around by hand, there is an automatic code fix if you `ctrl + .` to wrap a client method in one of these with the correct names.

Reply:
I did. Good to know.

Reply:
Having said that, the method names changed over time, but the strings stayed unchanged.

Reply:
See the other comment about interpolated values using `nameof` to have your refactorings update these too.